From 55ad71112d3dda1656a724118b32afaba73d6dae Mon Sep 17 00:00:00 2001
From: Ian Henriksen <insertinterestingnamehere@gmail.com>
Date: Sat, 18 Oct 2014 23:03:34 -0600
Subject: [PATCH] BUG: Match exception declarations between
 scipy/io/matlab/streams.pyx and scipy/io/matlab/streams.pxd to prevent an
 error in recent versions of cython.

---
 scipy/io/matlab/streams.pyx | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/scipy/io/matlab/streams.pyx b/scipy/io/matlab/streams.pyx
index d74e424..0ca3fc5 100644
--- a/scipy/io/matlab/streams.pyx
+++ b/scipy/io/matlab/streams.pyx
@@ -201,7 +201,7 @@ cdef class ZlibInputStream(GenericStream):
         return (self._max_length == self._read_bytes) and \
                (self._buffer_size == self._buffer_position)
 
-    cpdef long int tell(self):
+    cpdef long int tell(self) except -1:
         return self._total_position
 
     cpdef int seek(self, long int offset, int whence=0) except -1:
@@ -304,7 +304,7 @@ cdef class FileStream(GenericStream):
             raise IOError('Failed seek')
         return ret
 
-    cpdef long int tell(self):
+    cpdef long int tell(self) except -1:
         return ftell(self.file)
 
     cdef int read_into(self, void *buf, size_t n) except -1:
-- 
2.3.1

